data/human/deep missions.txt

```diff
@@ -2822,6 +2822,8 @@ mission "Stone of our Fathers 5"
 		dialog `You have reached <planet>, but your escort with the space reserved for Thorleif and Ragnhild has not arrived! Better depart and wait for your escorts to arrive in this star system.`
 	on complete
 		event "stone waiting most" 30
+		action
+			set "stone waiting most: triggered"
```

Comment:
You only need the `action` in a conversation.
The `on complete` is already an action.
```suggestion
		set "stone waiting most: triggered"
```